Enterprise-CMCS/macpro-mdct-mcr
# add first page reporting scenarios

### Description

Add/update state information and reporting scenario page fields

[Figma](https://www.figma.com/design/QcoVlfZ2adUBnzcHiI0G0S/MDCT-MCR-NAAAR?node-id=583-57657&t=67YJMnOwGRSyBKN5-1)

See notes :)

### Related ticket(s)


CMDCT-4097

---
### How to test

- Run app or view [deployed instance](https://d3vfr6zvnjvlnw.cloudfront.net/)
- Log in as a state user
- Create a NAAAR
- Enter the report and verify the first page matches the [figma](https://www.figma.com/design/QcoVlfZ2adUBnzcHiI0G0S/MDCT-MCR-NAAAR?node-id=583-57657&t=67YJMnOwGRSyBKN5-1)
- Fill out the page and verify it all works as expected (play around with nested "other text" field)


### Notes


The line separating fields from the footer buttons doesn't show up 🤔 

Also the spacing is not exactly what's in the design. anyone think it's worth having the exact css in that hint text?

---
### Pre-review checklist


---
### Pre-merge checklist


#### Review
---
